src/main/java/handlers/SimpleGetHandler.java

```diff
@@ -0,0 +1,21 @@
+package main.java.handlers;
+
+import main.java.server.Response;
+import main.java.server.request.Request;
+
+
+import static main.java.server.HTTPMessageComponents.HTTPSyntax.*;
+import static main.java.server.HTTPMessageComponents.StatusCodes.OK;
+
+public class SimpleGetHandler implements Handler {
+
+    @Override
+    public Response generateResponseForRequest(Request request) {
+        return new Response(VERSION + SP + OK + CRLF);
```

Comment:
If this constructor is always being called with `VERSION + SP + #{contentThatVaries} + CRLF`, maybe we want to abstract out the content that's constant?